docs/devGuide/projectManagement.md

```diff
@@ -139,12 +146,14 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
 
    <box type="info" seamless>
 
-   We do this for each MarkBind release (rather than require contributors to update the bundles for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
+   * We do this for each MarkBind release (rather than require contributors to update the bundles for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
+   * Typically, the bundle will be updated automatically in the expected test site's folders after running `npm run updatetest` in the previous step.
    </box>
 
 1. **Combine** the changes that you made in the last two steps with the [version commit](https://docs.npmjs.com/cli/version) generated by `lerna version`. To do so, amend the version commit and version tag by doing the following — take note to change `vA.B.C` to the new version.
 
    ```sh
+   $ git add .
```

Comment:
I believe the command `git commit -a` below would already automatically stage the modified files? There shouldn't be any new untracked files created during release so there is no need to explicitly run `git add`?

Reply:
Removed, thanks